docs/src/pages/components/popover/AnchorPlayground.js

```diff
@@ -99,9 +99,10 @@ function AnchorPlayground(props) {
   };
 
   const handleNumberInputChange = key => event => {
+    const value = event.target.value;
```

Comment:
I think this is better:
```const {value} = event.target;```

Reply:
I don't think that it makes a difference. I believe that the convention in the codebase is to not destructure the two options are equivalent.

Reply:
How did this PR fix a bug? As far as I can see, there is no functional difference in the new code and old code; if `target` is null, it will still be dereferenced and cause a crash.

Reply:
https://github.com/mui-org/material-ui/issues/20175

Reply:
Yeap, let's change the approach this time!